tpgtools/overrides/compute/samples/networkfirewallpolicyrule/global_update.tf.tmpl

```diff
@@ -0,0 +1,41 @@
+resource "google_compute_network_firewall_policy" "basic_network_firewall_policy" {
+  name = "{{policy}}"
+  project = "{{project}}"
+  description = "Sample global network firewall policy"
+}
+
+resource "google_compute_network_firewall_policy_rule" "primary" {
+ firewall_policy = google_compute_network_firewall_policy.basic_network_firewall_policy.name
+ action = "deny"
+ direction = "EGRESS"
+ priority = 1000
+match {
+layer4_configs {
+ip_protocol = "tcp"
+ports = ["123"]
+ }
+ dest_ip_ranges = ["0.0.0.0/0"]
+ }
+  target_secure_tags {
+ name = "tagValues/${google_tags_tag_value.basic_value.name}"
```

Comment:
Same as src tags- this seems like it's probably `Required`, but is currently marked as `Optional`.

Reply:
fixed in latest DCL